{{cookiecutter.project_name}}/pyproject.toml

```diff
@@ -168,12 +115,10 @@ exclude = [
     "**/settings/*",
     "snapshots",
 ]
-max-arguments = 12
 max-imports = 20
 max-import-from-members = 10
 max-expressions = 12
 max-methods = 10
-max-local-variables = 16
```

Comment:
Duplicated in Pylint - `max-locals`

Reply:
Yes, but flake8 will raise an error by this rule. You may check it localy

Reply:
`max-local-variables` config option is available for [wemake](https://wemake-python-styleguide.readthedocs.io/en/latest/pages/usage/configuration.html). It is not default config option - [doc search](https://flake8.pycqa.org/en/latest/search.html?q=max-local-variables&check_keywords=yes&area=default#)